scripts/cmake/ElektraCache.cmake

```diff
@@ -69,12 +69,12 @@ if (TOOLS MATCHES "DEFAULT")
 endif ()
 
 if (TOOLS MATCHES "NODEP")
-	set (TOOLS_LIST)
+	set (TOOLS_LIST race)
 	set (TOOLS_FORCE FORCE)
 endif ()
 
 if (TOOLS MATCHES "ALL")
-	set (TOOLS_LIST pythongen race qt-gui gen-gpg-testkey fuse)
+	set (TOOLS_LIST elektrad fuse gen-gpg-testkey hub-zeromq kdb pythongen qt-gui race webd website webui)
```

Comment:
I don't think building e.g. the website will work everywhere. I think there is only one docker image containing everything needed, and it makes sense to only build the website once in the pipeline.

Also: elektrad, webd, webui could be problematic on some images, but I am not sure here.

For the others: let's see if it will work 👍